testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -441,6 +441,20 @@ def test_universe_guess_bonds_off(self):
             assert not hasattr(u, attr)
         assert not u.kwargs['guess_bonds']
 
+    def test_fudge_factor(self):
+        """Test if fudge_factor parameter is being passed correctly"""
+        u = mda.Universe(two_water_gro, guess_bonds=True, fudge_factor=0.5)
+        self._check_universe(u)
+        assert u.kwargs["guess_bonds"]
+        assert_equal(0.5, u.kwargs["fudge_factor"])
+
+    def test_lower_bound(self):
+        """Test if lower_bound parameter is being passed correctly"""
+        u = mda.Universe(two_water_gro, guess_bonds=True, lower_bound=0.8)
+        self._check_universe(u)
+        assert u.kwargs["guess_bonds"]
+        assert_equal(0.8, u.kwargs["lower_bound"])
```

Comment:
There's really no point of separating this out into two separate tests, I would just do the one test that checks that kwargs is populated fine.

Also here, due to `assert_equal`'s very large cost on single values, I would encourage you to use a standard assert instead, you can use pytest.approx for comparing floats, see: https://github.com/MDAnalysis/mdanalysis/blob/a05a5245042361fd4fa27f03993f34cee2cda901/testsuite/MDAnalysisTests/analysis/test_align.py#L569